docs/_includes/_joiningBlocks/footer/10-menu.njk

```diff
@@ -0,0 +1,40 @@
+<div id="footer-menu">
+  <div class="content-area">
+    {% for category in footer %}
+      <nav>
+        <h3>{{category.name}}</h3>
+        <ul>
+          {% for entry in category.children %}
+            <li>
+              <a href="{{entry.href | url}}">{{entry.text}}</a>
+            </li>
+          {% endfor %}
+        </ul>
+      </nav>
+    {% endfor %}
+
+    <div id="footer-badges">
+
+        <a rel="license" href="https://creativecommons.org/publicdomain/zero/1.0/">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="-0.5 -0.101 88 31">
```

Comment:
```suggestion
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="-0.5 -0.101 88 31" width="100" height="35">
```

It's good for SVGs to include width and height attributes so they don't oversize on page load if there's an error or delay in loading the CSS.